extensions/wikia/ArticleVideo/bower_components/html5-skin/js/components/controlBar.js

```diff
@@ -412,23 +426,39 @@ var ControlBar = React.createClass({
 
     for (var k = 0; k < defaultItems.length; k++) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the total number of "break" and "continue" statements in this loop to use one at most. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ATooManyBreakOrContinueInLoop)